Gemfile

```diff
@@ -40,12 +40,8 @@ group :test do
       gem 'oj' unless RUBY_VERSION == '1.9.3'
     end
 
-    # Ox after 2.4.12 has a change in default behavior. Test both.
-    if ENV['OLD_OX']
-      gem 'ox', '2.4.12'
-    else
-      gem 'ox'
-    end
+    # As ox suggestion
+    gem 'ox', '~> 2.8.1'
```

Comment:
This is possibly too specific - if regressions appear in new 'ox' versions, we will miss it with this version lock. If we want, this could be the "new" `OLD_OX` and we keep the unbounded dependency for later tests.